cfme/configure/access_control.py

```diff
@@ -292,7 +295,22 @@ def retrieve_ldap_user_groups(self):
                                'username': self.ldap_credentials.principal,
                                'password': self.ldap_credentials.secret,
                                },
-             action=form_buttons.retrieve)
+             )
+        # adding sleep due to JS issue
+        import time
+        time.sleep(3)
+        sel.click(form_buttons.retrieve)
+
+    def retrieve_ext_auth_user_groups(self):
```

Comment:
This is OK for now, but in the future this should be a ```_function_name``` because it shouldn't ever be called by a user